### PR TITLE
register driver and operators design

### PR DESCRIPTION
**Pull request by UmuhireJessie**
make suggestions on the design

### CODE REVIEW

**Review by fab-ryan — APPROVED** [review]

**Review by aimelive — APPROVED** [review]

**Review by GilbertUp — APPROVED** [review]

**Review by GilbertUp — APPROVED** [review]
resolve conflicts!

**Comment by Anguandia** [comment]
@UmuhireJessie , please get in touch with @furebo to resolve these conflicts, then squash your commits

File: src/components/admin-header/Header.css [hunk 1]
```diff
@@ -221,7 +117,75 @@ a.d-icons p{
...
+    .sideBar nav a.active {
+        color: var(--theme-green);
+    }
+    .sideBar nav a.active:hover {
+        color: var(--theme-sideBar-white);
+    }
+    .mobileMainContent {
+        min-height: 86vh;
+    }
+=======
```
Comment (furebo) [inline src/components/admin-header/Header.css]: Please resolve these conflicts

File: src/components/admin-sidebar/SideBar.js [hunk 2]
```diff
@@ -43,20 +43,29 @@ const Sidebar = () => {
...
+=======
+>>>>>>> Added roles and permissions
+=======
+        </NavLink>
+>>>>>>> worked on issue #26
         <NavLink to="/crud-route">
           <AiIcons.AiOutlineReload />
           <span> Routes </span>
         </NavLink>
+<<<<<<< HEAD
```
Comment (furebo) [inline src/components/admin-sidebar/SideBar.js]: Remove the conflicts also in this file

File: webpack.prod.js [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
 const CssMinimizerPlugin = require("css-minimizer-webpack-plugin");
 const TerserPlugin = require("terser-webpack-plugin");
+=======
+>>>>>>> worked on issue #26
 const common = require("./webpack.common.js");
 
 module.exports = merge(common, {
   mode: "production",
   devtool: "source-map",
+<<<<<<< HEAD
```
Comment (furebo) [inline webpack.prod.js]: the same here remove the conflicts

File: webpack.prod.js [hunk 4]
```diff
@@ -1,12 +1,16 @@
 const { merge } = require("webpack-merge");
+<<<<<<< HEAD
```
Comment (furebo) [inline webpack.prod.js]: Remove conflicts in this file

File: webpack.dev.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ const common = require("./webpack.common.js");
 module.exports = merge(common, {
   mode: "development",
   devtool: "inline-source-map",
+<<<<<<< HEAD
```
Comment (furebo) [inline webpack.dev.js]: Please remove conflicts

**Review by furebo — CHANGES_REQUESTED** [review]

File: src/components/admin-header/Header.css (hunk 1 above)
Comment (tjeaneric) [reply src/components/admin-header/Header.css]: Resolved now.

File: src/components/admin-sidebar/SideBar.js (hunk 2 above)
Comment (tjeaneric) [reply src/components/admin-sidebar/SideBar.js]: Resolved now.

File: webpack.prod.js (hunk 3 above)
Comment (tjeaneric) [reply webpack.prod.js]: Resolved now.

File: webpack.prod.js (hunk 4 above)
Comment (tjeaneric) [reply webpack.prod.js]: Resolved now.

File: webpack.dev.js (hunk 5 above)
Comment (tjeaneric) [reply webpack.dev.js]: Resolved now.

**Review by furebo — APPROVED** [review]

**Pull request merged**